packages/aws-client/tests/test-KMS.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const test = require('ava');
+const { kms } = require('../services');
+const KMS = require('../KMS');
+
+test.before(async (t) => {
+  const createKeyResponse = await kms().createKey({}).promise();
+  t.context.KeyId = createKeyResponse.KeyMetadata.KeyId;
+});
+
+test('createKey() creates a key', async (t) => {
+  const createKeyResponse = await KMS.createKey();
+
+  await t.notThrowsAsync(
+    kms().describeKey({
+      KeyId: createKeyResponse.KeyMetadata.KeyId
+    }).promise()
+  );
+});
+
+test('encrypt() properly encrypts a value', async (t) => {
+  const ciphertext = await KMS.encrypt(t.context.KeyId, 'asdf');
+
+  const plaintext = await kms().decrypt({
```

Comment:
I assume you're not using `KMS.decryptBase64String` here so that the test is only unit testing one function

Reply:
Correct.